Alignment/OfflineValidation/python/TkAlAllInOneTool/Zmumu_cfg.py

```diff
@@ -194,4 +194,18 @@
             closeFileFast = cms.untracked.bool(True),
     )
 
-process.p = cms.Path(process.offlineBeamSpot*process.TrackRefitter*process.DiMuonMassValidation)
+###################################################################
+# Beamspot compatibility check
+###################################################################
+from RecoVertex.BeamSpotProducer.beamSpotCompatibilityChecker_cfi import beamSpotCompatibilityChecker
+process.BeamSpotChecker = beamSpotCompatibilityChecker.clone(
+    bsFromEvent = "offlineBeamSpot::RECO",  # source of the event beamspot (in the ALCARECO files)
+    bsFromDB = "offlineBeamSpot",           # source of the DB beamspot (from Global Tag) NOTE: only if dbFromEvent is True!
+    warningThr = cms.double(config["validation"].get("bsIncompatibleWarnThresh", 3)), # significance threshold to emit a warning message
+    errorThr = cms.double(config["validation"].get("bsIncompatibleErrThresh", 5)),    # significance threshold to abort the job
```

Comment:
Explicit type could also get avoided here, I guess
```suggestion
    warningThr = config["validation"].get("bsIncompatibleWarnThresh", 3), # significance threshold to emit a warning message
    errorThr = config["validation"].get("bsIncompatibleErrThresh", 5),    # significance threshold to abort the job
```